src/multicloud-connector/HISTORY.rst

```diff
@@ -0,0 +1,8 @@
+.. :changelog:
+
+Release History
+===============
+
+1.0.0b1
+++++++
```

Comment:
` az arc-multicloud generate-aws-template ` outputs the aws cloudformation template as is which is of no use for customer consumption. It's not a valid aws cloudformation template.

Please add logic for the below items,
1. Register the required RPs (Microsoft.AwsConnector, Microsoft.HybridConnectivity, Microsoft.HybridCompute).
2. From the response body, remove the "body:" as it's not a valid aws cloudformation template.
3. Write the aws cloudformation template to a file and output the file on to the console.
4. remoe the `--solution-types` parameter to avoid unnecessary confusion to customers. Help message doesn't show what should be provided in the `--solution-types` parameter. This was added mainly for the portal onboard flow. For CLI, we can assume that it's not supported scenario.
5. The examples should be more meaningful rather than some random strings,
![image](https://github.com/user-attachments/assets/f4afc39a-0698-4f1f-b6b8-29084f9880f6)